### PR TITLE
Update to 6.5.3

### PR DESCRIPTION
**Pull request by marcelotduarte**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

@conda-forge-admin, please rerender

### CODE REVIEW

**Comment by marcelotduarte** [comment]
@conda-forge-admin, please rerender

**Comment by marcelotduarte** [comment]
@conda-forge-admin, please rerender

**Comment by jschueller** [comment]
I propose to just skip osx+py39

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
+  - 10.15                  # [osx and x86_64]
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - 10.15                  # [osx and x86_64]
```
Comment (marcelotduarte) [inline recipe/conda_build_config.yaml:4]: When I added this configuration, the errors decreased, before there were 20 errors ...

**Comment by marcelotduarte** [comment]
> I propose to just skip osx+py39

Can you do it?

**Comment by jschueller** [comment]
it's just a matter of adding "skip: true  # [osx and py39]" to the "build" section

**Pull request closed without merging**